tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/Tensor.java

```diff
@@ -212,4 +246,23 @@
    */
   @Override
   void close();
+
+  public static class ToStringOptions {
```

Comment:
What do you think about adding a float format option?  Either just the decimal places or a `String.format`/printf` style format.  Maybe use `String.format` internally?

Reply:
Good idea, but I'm going to leave this for someone else to add as a future PR.